### ISSUE TITLE
Does not check off problems on Opera GX

### ISSUE THREAD

**Issue by eightteababi** [The-CodingSloth/haha-funny-leetcode-extension#65]
The extension is cool, but after I have completed the problem on my browser Opera GX, it is not marked as completed by the extension, so I will be stuck inside Leetcode forever despite having successfully tortured myself :0

**Comment by eightteababi** [comment]
Huhm looks like this is a problem with the new Dynamic Layout of Leetcode. I read other comments and a new update is probably on the way right?

**Comment by The-CodingSloth** [comment]
yup trying to get this fixed and trying to get the firefox version fully supported. I'm not sure why this bug is happening, it happens randomly so I'll try my best, but if anyone else figures it out feel free to fix it haha

**Comment by eightteababi** [comment]
Update: Tried to fix the problem by switching to the "old version" setting of Leetcode. and yes it still doesn't work lmao (of course it won't be that easy). 
Will try to figure out the issue but I have to do my assignment first :')

**Comment by The-CodingSloth** [comment]
Added a possible fix in this new development build, if you could test it out and let me know if that worked, I'd appreciate it. Only tested it on Chrome and Brave.

**Comment by AbdullahDaGoat** [comment]
I tested it out and am using operaGx, it didnt work here are my specs
Browser identification
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36 OPR/101.0.0.0

**Comment by The-CodingSloth** [comment]
Alright I just pushed a change in the code, can anyone test this out like a bunch of times. just resave the code to generate a new problem. I'm hoping this fixed it or i'm gonna go crazy.
